connectors/gridgo-mysql/src/main/java/io/gridgo/connector/mysql/JdbcTransaction.java

```diff
@@ -0,0 +1,92 @@
+package io.gridgo.connector.mysql;
+
+
+import io.gridgo.connector.impl.AbstractProducer;
+import io.gridgo.connector.mysql.support.JdbcOperationException;
+import io.gridgo.connector.support.config.ConnectorContext;
+import io.gridgo.framework.support.Message;
+import lombok.extern.slf4j.Slf4j;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.transaction.TransactionException;
+import org.joo.promise4j.Promise;
+import org.joo.promise4j.impl.CompletableDeferredObject;
+import org.joo.promise4j.impl.SimpleFailurePromise;
+
+
+import static io.gridgo.connector.mysql.JdbcConstants.*;
+
+@Slf4j
+class JdbcTransaction extends JdbcClient {
+    private AbstractProducer producer;
+
+    JdbcTransaction(Handle handle, ConnectorContext context) {
+        super(context);
+        this.handle = handle;
+    }
+
+    private final Handle handle;
+
+    @Override
+    protected String generateName() {
+        return null;
+    }
+
+    @Override
+    protected Promise<Message, Exception> _call(Message request, CompletableDeferredObject<Message, Exception> deferred, boolean isRPC) {
+        var operation = request.headers().getString(OPERATION);
+        var handler = operationsMap.get(operation);
+        if (handler == null) {
+            return new SimpleFailurePromise<>(new JdbcOperationException());
+        }
+        try {
+            Message result = handler.handle(request, handle);
```

Comment:
use execution strategy here (same as above). can move this common code to the abstract class